specs/conductor.md

```diff
@@ -1,59 +1,137 @@
 # Astria Conductor
 
-The Astria conductor connects the shared sequencer and DA layers to the execution layer. When a block is received from the sequencer layer or DA layer, the conductor filters out the transactions that are relevant to the rollup's namespace and pushes it to the execution layer. 
+The Astria Conductor connects the shared Sequencer and data availability layers to the execution layer. There is one instance of the Conductor per rollup.
```

Comment:
```suggestion
The Astria Conductor connects the shared sequencer and data availability layers to the execution layer. There is one instance of the Conductor per rollup node.
```

Reply:
not even sure if "rollup node" is the right term here, it's more like "rollup execution environment" while "rollup node" in our stack would refer to the combo of "rollup execution environment + conductor"